.github/workflows/deploy.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy to Amazon ECR
+
+env:
+  aws_region: eu-west-2
+  ecr_repository: ${{ github.event.repository.name }}
+  dockerfile: "Dockerfile"
+
+on:
+  push:
+    branches: [master]
```

Comment:
Consider changing this to `master` and `main` to future proof. Or the new {{default branch}} or whatever it's called

Reply:
> Or the new {{default branch}} or whatever it's called

Yea I misread. I'd imagine once theres a change we'd want to mass-change it, so no need to update it now.